files/chef-marketplace-cookbooks/chef-marketplace/recipes/_runit_prepare.rb

```diff
@@ -0,0 +1,27 @@
+include_recipe 'enterprise::runit'
+
+usr = grp = node['chef-marketplace']['runit']['user']['username']
+
+directory '/var/opt/chef-marketplace' do
+  recursive true
+  owner usr
+  group grp
+  mode '0775'
+end
+
+directory '/var/log/chef-marketplace' do
+  recursive true
+  owner usr
+  group grp
+  mode '0775'
+end
+
+user usr do
```

Comment:
Should this and the `group` resource below be moved above the directories so they exist before the directories are created?